index.css

```
@@ -0,0 +1,35 @@
+body {
+  font-family: 'monospace';
+  font-weight: 400;
+  color: black;
+  line-height: 1.2;
+  font-size: 0.9rem;
```

The font appears a bit small for me, can we use 1.0 or atleast 0.95 something?

```suggestion
  font-size: 1.0rem;
```